smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -48,6 +48,7 @@ be used.
                             Sockets, otherwise N/A
                             switch form uses local IP address as hostname
   --username                anonymous, or Key Vault if --registry is specified
+  --no-deployment           Start IoT Edge modules deployment (EdgeHub and custom modules)
```

Comment:
I initially read this as "--no-deployment means Start IoT Edge modules deployment", but it makes sense when you realize starting a deployment is the default.  Would it make it more clear to reword to something like, "Flag not set, start IoT Edge modules deployment"?

Reply:
I'm not too happy with how this whole help message reads, but for now this is the "Defaults" sections, so it's a list of behaviors when the various flags/parameters are _not_ set. So "flag not set" is implied.

Reply:
Changed to "deploy Edge Hub and temperature sensor modules", as "not set" is implicit

---
In reply to: [203526314](https://github.com/Azure/iotedge/pull/37#discussion_r203526314) [](ancestors = 203526314)